lib/improver/tests/utilities/test_temporal.py

```diff
@@ -452,6 +480,43 @@ def test_invalid_date_format(self):
                                forecast_time=6)
 
 
+class Test_extract_nearest_time_point(IrisTest):
+
+    """Test the extract_nearest_time_point function."""
+
+    def setUp(self):
+        """Set up a cube for the tests."""
+        cube = set_up_cube(num_time_points=2)
+        self.cube = add_forecast_reference_time_and_forecast_period(
+            cube, time_point=[402295.0, 402296.0], fp_point=[4.0, 5.0])
+
+    def test_time_coord(self):
```

Comment:
It might be nice to show what happens in a tied case, in this case an input datetime of 2015,11,23,7.5 in a unit test.

Reply:
Added a couple of unit tests to show the behaviour, to the nearest minute.